### ISSUE TITLE
how to get the best LR

### ISSUE THREAD

**Issue by ardeal** [davidtvs/pytorch-lr-finder#84]
Hi,
I run the following code and got the following picture.
I am wondering what the best LR is in my picture?
how to print/display the best LR in the code?

```python

      finding_lr = lr_finder.LRFinder(net.model, optimizer, criterion, device="cpu")

        finding_lr.range_test(train_loader, start_lr=0.0001, end_lr=0.1, num_iter=1000, step_mode="exp")
    
        finding_lr.plot(savefig_path=os.path.join(args.output_dir, 'lr_finder.png'))
```

![image](https://user-images.githubusercontent.com/6227348/163792468-85b863e4-5ae4-4c27-98f6-7d6674298144.png)

**Comment by NaleRaphael** [comment]
Hi @ardeal,

You can get a suggested learning rate by passing argument `suggest_lr` to `lr_finder.plot()`, it will return an axis of plot and a suggested lr.

**Comment by ardeal** [comment]
@NaleRaphael ,
You argument `suggest_lr` works. Thanks!
